src/sources/aws_ecs_metrics/parser.rs

```diff
@@ -332,46 +350,79 @@ fn memory_metrics(
     metrics.extend(
         vec![
             ("used_bytes", memory.usage),
-            ("max_used_bytes", memory.max_usage),
+            ("max_used_bytes", memory.max_usage.unwrap_or_default()),
```

Comment:
This will end up using `0`, right? I wonder if we should avoid publishing the metric at all if we weren't able to scrape it.